### PR TITLE
fix: warn user when using unencrypted non-local urls

### PR DESCRIPTION
**Pull request by adamewozniak**
Adds a log warning whenever unencrypted non-local urls are used in flags. Since it uses `url.Parse`, it should also give an error if unable to parse the url. 

Closes : #178

### CODE REVIEW

File: cmd/peggo/flags.go [hunk 1]
```diff
@@ -103,3 +107,14 @@ func bridgeFlagSet() *pflag.FlagSet {
 
 	return fs
 }
+
+// parseURL logs a warning if the flag provided is an
+// unencrypted non-local string, and returns the value.
+// Ref: https://github.com/umee-network/peggo/issues/178
+func parseURL(logger zerolog.Logger, konfig *koanf.Koanf, flag string) string {
+	endpoint := konfig.String(flag)
+	if strings.Contains(endpoint, "http://") && !strings.Contains(endpoint, "localhost") {
```
Comment (alexanderbez) [inline cmd/peggo/flags.go:116]: why not call `url.Parse` and check the `Protocol` and `Hostname` instead??

**Review by alexanderbez — APPROVED** [review]

**Comment by alexanderbez** [comment]
@adamewozniak let's backport this :)

**Comment by alexanderbez** [comment]
Also, we're missing a changelog entry for this. I'm doing a poor job catching this in PRs, but we need to make sure we keep a tidy and useful changelog.

**Comment by adamewozniak** [comment]
@Mergifyio backport release/v0.2.x

**Pull request merged**